examples/set-a-custom-referrer/README.md

```diff
@@ -0,0 +1,27 @@
+# Set a Custom Referrer
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [TestCafe Example Page](https://devexpress.github.io/testcafe/example/)
+
+The [HTTP referrer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) (otherwise known as referer) is a request header that lets a webpage know where the user has navigated from.
+
+This example shows how to set a custom referrer for the specified requests during tests.
+
+Before the test begins a request hook and a request logger are introduced. When requests are made to a webpage, the hook overwrites the requests's 'Referer' header with a desired value.
```

Comment:
> When requests are made to a webpage,

You mean the new webpage? Hm, that's not quite clear. And requests **for** the webpage are made **to** the web server I guess.

Maye we can say: *When the browser sends requests,*

Reply:
> a request hook and a request logger are introduced.

Maybe just *created*. That's ok to say "introduce" meaning "create", but here it doesn't fit well. We're at the very beginning of an example and we're simply creating things.

Reply:
> Before the test begins a request hook and a request logger are created. When the browser sends requests, the hook overwrites the requests's 'Referer' header with a desired value.